packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -149,6 +149,16 @@ export interface IDataStore extends IFluidRouter {
 	 * the data store's entryPoint.
 	 */
 	readonly entryPoint?: IFluidHandle<FluidObject>;
+
+	/**
+	 * @deprecated - TODO: will be removed from interface in future major release
+	 */
+	request(request: IRequest): Promise<IResponse>;
```

Comment:
This is an iffy one since it is technically used in Loop in one place (that one place in can just ignore the deprecation for now).

Reply:
It seems fine to me to deprecate this. What do you mean by `hat one place in can just ignore the deprecation for now`? The usage in Loop to get the underlying object?